src/dashboard/get_chart_data/templates/filter_numeric.sql.jinja

```diff
@@ -0,0 +1,48 @@
+{%- import 'syntax.sql.jinja' as syntax -%}
+
+{%- set ns =namespace(matches=[]) -%}
+{%- set filters=['eq', 'ne', 'gt', 'gte', 'lt', 'lte'] -%}
+
+{%- macro render_filter( data, filter_type, bound) -%}
+{%- if filter_type == 'eq'-%}
+{{ data }} = {{ bound }}
```

Comment:
Add double quotes around these column references?

Reply:
Removed this file, but generally did passes to ensure fields were properly quoted